sources/packages/backend/apps/api/src/route-controllers/student/student.external.controller.ts

```diff
@@ -90,10 +91,20 @@ export class StudentExternalController extends BaseController {
         studentApplications,
         legacyApplications,
       );
+    // Log found student and SFAS individual IDs.
+    if (student) {
+      this.logger.log(`Found SIMS student with ID: ${student.id}`);
+    }
+    if (sfasIndividual) {
```

Comment:
The same condition, if(sfasIndividual) is at line 82 and we can make use of the same instead, adding the log for legacy after line 86.